Makefile

```diff
@@ -1,3 +1,5 @@
+CC = gcc
```

Comment:
`CC` will default to `"cc"`, which is usually symlinked to the system's default C compiler. We shouldn't force it to be GCC.

Reply:
Thanks, removed and commit amended.